Doc/library/functools.rst

```diff
@@ -379,12 +379,15 @@ The :mod:`functools` module defines the following functions:
    :data:`!Placeholder` sentinel to the place held by previous :data:`!Placeholder`:
 
       >>> from functools import partial, Placeholder as _
-      >>> show5 = partial(print, _, _, _, 4)
-      >>> show5 = partial(show5, _, _, 3)
-      >>> show5 = partial(show5, _, 2)
-      >>> show5 = partial(show5, _, 5)   # 5 is appended after 4
-      >>> show5(1)
-      1 2 3 4 5
+      >>> remove = partial(str.replace, _, _, '')
+      >>> remove('Hello, world', 'l')
+      'Heo, word'
+      >>> remove_l = partial(remove, _, 'l')
```

Comment:
Actually, this may be not so good example, because `l` can be confused with `1` or `I`. Could you try to find better string for example?

`str.replace` is perhaps the best example, because we need a function that accepts at least 3 (preferably 4) positional arguments.

Reply:
Will do. Nitpicks are good, just maybe not very productive for that specific area that is going to change soon. Thanks!